docs/source/ipyleaflet.ipynb

```diff
@@ -8,7 +8,7 @@
    "outputs": [],
    "source": [
     "import piplite\n",
-    "await piplite.install('ipyleaflet')"
+    "await piplite.install('ipyleaflet == 0.17')"
```

Comment:
Maybe the extra spaces around the `==` could be trimmed?

```suggestion
    "await piplite.install('ipyleaflet==0.17')"
```

Reply:
Thanks.This has been changed.